lib/agent.js

```diff
@@ -244,6 +244,7 @@ Agent.prototype.captureError = function (err, opts, cb) {
     error.parent_id = context.id
     error.trace_id = context.traceId
     error.timestamp = timestamp
+    error.sampled = trans && trans.sampled
```

Comment:
I think `trans` will be `null` if there's no active transaction, in which case the `sampled` property wrongly will be set to `null` as well

Reply:
fixed.